shapely/geos.py

```diff
@@ -414,6 +414,12 @@ def errcheck_just_free(result, func, argtuple):
     else:
         return retval
 
+def errcheck_null_exception(result, func, argtuple):
+    '''Wraps errcheck_just_free, raising a TopologicalError if result is NULL'''
+    if not result:
+        raise TopologicalError("The operation '{0}' could not be performed. \
```

Comment:
Sure. I'm aware of this convention in Shapely, although the little group of functions around that section don't follow it. I might as well correct them all while I'm in there.